packages/cursorless-engine/src/cursorlessEngine.ts

```diff
@@ -63,10 +65,8 @@ export async function createCursorlessEngine(
   const languageDefinitions = new LanguageDefinitions(fileSystem, treeSitter);
   await languageDefinitions.init();
 
-  const talonSpokenForms = new TalonSpokenFormsJsonReader(fileSystem);
-
   const customSpokenFormGenerator = new CustomSpokenFormGeneratorImpl(
-    talonSpokenForms,
+    talonSpokenForms ?? new DisabledTalonSpokenForms(),
```

Comment:
did you try forcing this to use disabled and seeing if cursorless side bar reverts to default spoken forms?

Reply:
You were actually right. Returning an empty list made it so that all spoken forms were disabled :D
I made a commit to support this new usecase